analyzers/pastebinurlanalyzer.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from .urlanalyzer import URLAnalyzer
+
+
+class PastebinURLAnalyzer(URLAnalyzer):
+    """ Analyzer to match Pastebin URLs via regex """
+    name = "PastebinURLAnalyzer"
+
+    def __init__(self, action, resolve=False):
+        regex = "((?:https?:\/\/)?pastebin.com\/\S+)"
+        super().__init__(action, regex, resolve)
```

Comment:
Missing newline, but I'll add it later :)

Also nice that you subclassed URL Analyzer as written in the issue.